GenerateToolingFeed/Program.cs

```diff
@@ -66,50 +66,55 @@ private static bool TryUpdateFeedWithNewToolsAndTemplates(JObject feed, FeedForm
         {
             try
             {
-                // Get a cloned object to not modify the exisiting release
-                JObject currentReleaseEntryJson = GetCurrentReleaseEntry(feed, coreToolsInfo.MajorVersion).DeepClone() as JObject;
-                JObject newReleaseEntryJson = GetNewReleaseEntryJson(currentReleaseEntryJson, format, coreToolsInfo);
-                return TryAddNewReleaseToFeed(feed, newReleaseEntryJson, coreToolsInfo.MajorVersion);
+                List<string> tags = Helper.GetTagsFromMajorVersion(coreToolsInfo.MajorVersion);
+
+                bool result = true;
+                foreach (string tag in tags)
+                {
+                    string releaseVersion = Helper.GetReleaseVersionFromTag(feed, tag);
+
+                    // Get a cloned object to not modify the exisiting release
+                    JObject currentReleaseEntryJson = feed["releases"][releaseVersion].DeepClone() as JObject;
+
+                    JObject newReleaseEntryJson = GetNewReleaseEntryJson(currentReleaseEntryJson, format, coreToolsInfo, tag);
+
+                    result &= TryAddNewReleaseToFeed(feed, newReleaseEntryJson, coreToolsInfo.MajorVersion, tag);
+                }
+                return result;
             }
             catch
             {
                 return false;
             }
         }
 
-        private static JObject GetNewReleaseEntryJson(JObject currentReleaseEntry, FeedFormat format, CoreToolsInfo coreToolsInfo)
+        private static JObject GetNewReleaseEntryJson(JObject currentReleaseEntry, FeedFormat format, CoreToolsInfo coreToolsInfo, string tag)
         {
-            IFeedEntryUpdater feedEntryUpdater = FeedEntryUpdaterFactory.GetFeedEntryUpdater(format);
+            IFeedEntryUpdater feedEntryUpdater = FeedEntryUpdaterFactory.GetFeedEntryUpdater(format, tag);
             return feedEntryUpdater.GetUpdatedFeedEntry(currentReleaseEntry, coreToolsInfo);
         }
 
-        private static bool TryAddNewReleaseToFeed(JObject feed, JObject newRelease, int majorVersion)
+        private static bool TryAddNewReleaseToFeed(JObject feed, JObject newRelease, int majorVersion, string tag)
         {
             JObject feedReleases = feed["releases"] as JObject;
-            string newReleaseVersion = Helper.GetReleaseVersion(feedReleases, majorVersion);
+            string newReleaseVersion = Helper.GetNewReleaseVersion(feedReleases, majorVersion);
 
             if (newReleaseVersion == null)
             {
                 return false;
             }
 
+            if (tag.StartsWith("v0", StringComparison.OrdinalIgnoreCase))

```

Comment:
similar comment to the one above. Hardcoding this here will make maintenance much more difficult down the road.

Reply:
Moved this to config as well